Readme.md

```diff
@@ -29,9 +29,10 @@ jobs:
         distribution: 'temurin'
         java-version: 21
         
-    - uses: gradle/actions/setup-gradle@v3
-      with:
-        arguments: dependencies
+    - uses: gradle/actions/setup-gradle@v4
+
+    - name: Run dependencies
+      run: ./gradlew dependencies
```

Comment:
Acutally this shouldn't be needed anymore 👍 I don't remember why this was needed in the first place, but i can be safely removed now.